### PR TITLE
VDS-1497: Prepare for 1.0 release

### PR DESCRIPTION
**Pull request by OlegoO**
Description: 
Added basic information about the module, connected mkdocs, logo.

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/VirtoCommerce.SimpleExportImport_0.0.1-preview-pr-40-f660f91b.zip

### CODE REVIEW

File: docs/BA docs/Registration.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+### Main entities
+
+Accounts data are managed in the Contact module. Native Virto Commerce functionality contains the following type of accounts and contacts:
+
+Type | Description | 
+--- | --- 
+Organization | Represents a buyer organization in the system, an account (in business terms) in B2B case. Is a container for other organizations (brunches or sub-companies) and contacts.
+Contact | Represents a physical body that can be an account (in business terms) by himself in B2C case or may belong to one or more organization accounts in B2B case. 
+Vendor | Represents a vendor organization in the system, can not contain other entities (vendors or contacts) by default.
+Employee | Represnts the seller (system owner) employee. May belong to one or more organization accounts in B2B case.
```
Comment (pushnitsa) [inline docs/BA docs/Registration.md]: Repres**e**nts (typo)

File: docs/BA docs/Registration.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+### Indexation
+
+Organizations, contacts, vendors, and employees are kept in the index.
+
+
+## Atomic functions / scenarios
+ 
+
+ Name | Module | Description | Link
```
Comment (pushnitsa) [inline docs/BA docs/Registration.md]: What that for?

Comment (asvishnyakov) [reply docs/BA docs/Registration.md]: @OlegoO

File: docs/BA docs/Registration.md (hunk 1 above)
Comment (asvishnyakov) [reply docs/BA docs/Registration.md]: Fixed

File: docs/BA docs/Registration.md (hunk 2 above)
Comment (asvishnyakov) [reply docs/BA docs/Registration.md]: Removed as unrelated

**Review by pushnitsa — APPROVED** [review]

**Pull request merged**